src/Engine/ProtoCore/AssociativeGraph.cs

```diff
@@ -45,10 +45,7 @@ public static void MarkAllGraphNodesDirty(int block, List<AssociativeGraph.Graph
             {
                 foreach (AssociativeGraph.GraphNode gnode in graphNodesInScope)
                 {
-                    if (gnode.languageBlockId == block)
-                    {
-                        gnode.isDirty = true;
-                    }
+                    gnode.isDirty = true;
```

Comment:
maybe we can add a new function instead of reusing this one? The summary no longer makes sense, and the block parameter is not used...